input/collector.go

```diff
@@ -106,6 +107,7 @@ func (s *TCPCollector) Addr() net.Addr {
 }
 
 func (s *TCPCollector) handleConnection(conn net.Conn, c chan<- *Event) {
+	log.Println("Handling a Connection...")
```

Comment:
This log line isn't helpful, can you remove it? Or add more technical information (source IP). TBH it looks like a debug statement.